hydra/conf/hydra/callbacks/default.yaml

```diff
@@ -0,0 +1,3 @@
+# @package hydra
+
+callbacks: {}
```

Comment:
This can work as an empty file as well, right?

Reply:
Yes, it works. I've made the Yaml empty. :)